packages/map-template/src/components/Map/Map.jsx

```diff
@@ -60,14 +61,18 @@ function Map({ apiKey, gmApiKey, mapboxAccessToken, venues, venueName, onLocatio
     }, [venueName, venues]); // eslint-disable-line react-hooks/exhaustive-deps
     // We ignore eslint warnings about missing dependencies because mapsIndoorsInstance should never change runtime anyway.
 
-    /*
-     * Show the filtered locations on the map based on their IDs.
+    /**
```

Comment:
Not a function definition. No need for JSDoc.

```suggestion
    /*
```